disnake/state.py

```diff
@@ -1063,6 +1063,12 @@ def parse_channel_pins_update(self, data) -> None:
             utils.parse_time(data["last_pin_timestamp"]) if data["last_pin_timestamp"] else None
         )
 
+        try:
+            channel.last_pin_timestamp = last_pin  # type: ignore
+        except AttributeError:
+            # The channel doesn't support it.
+            pass
```

Comment:
Another way to do this is just an `isinstance` with the supported types, but I'm not sure which is better 🤔

Reply:
isinstance is better imo